golang/cosmos/x/vibc/types/ibc_module.go

```diff
@@ -4,6 +4,7 @@ import (
 	fmt "fmt"
 
 	sdkioerrors "cosmossdk.io/errors"
+	"github.com/Agoric/agoric-sdk/golang/cosmos/types"
```

Comment:
There are too many "types" packages for me to be comfortable with referencing any of them as just "types". We don't need to clean up all the old instances, but we should at least stop digging.
```suggestion
	agtypes "github.com/Agoric/agoric-sdk/golang/cosmos/types"
```

Reply:
👍